spring-integration-core/src/main/java/org/springframework/integration/support/leader/LockRegistryLeaderInitiator.java

```diff
@@ -400,6 +400,9 @@ else if (acquired) {
 							}
 							return null;
 						}
+						else if (logger.isDebugEnabled()) {
+							logger.debug("Error to acquire the lock. " + (isRunning() ? "Retrying..." : ""), e);
```

Comment:
Perhaps the error message could be worded like _Error acquiring the lock_? Maybe it's just me but _Error to acquire the lock_ sounds a bit confusing at first.

Reply:
:smile: 
Fine with me. English is far away from my fluent language, so I'll deffer the final wording polishing to @garyrussell 

Glad to hear that it is going to be helpful, @vpavic !

Thank you!